features/support/env.rb

```diff
@@ -4,6 +4,8 @@
 require 'cucumber/rspec/doubles'
 require 'capybara/poltergeist'
 require 'webmock/cucumber'
+require 'capybara-screenshot/cucumber'
```

Comment:
For some reason they were being saved to the root directory. I explicitly have them saved to `tmp` in the app now and I think that's in the .gitignore